AEPAssurance/UnitTests/AssuranceSessionOrchestratorTests.swift

```diff
@@ -247,11 +248,8 @@ class AssuranceSessionOrchestratorTests: XCTestCase {
         sessionOrchestrator.session = session
         sessionOrchestrator.createQuickConnectSession(with: sampleSessionDetails)
         
-        XCTAssertFalse(mockAuthorizingPresentation.sessionConnectingCalled)
-        
-        XCTAssertFalse(mockStateManager.shareAssuranceStateCalled)
+        XCTAssertTrue(mockStateManager.clearAssuranceStateCalled)
```

Comment:
nit: Do you also want to verify the state of `outboundEventBuffer` ?

Reply:
Good call. Added